contrib/go-chi/chi/chi.go

```diff
@@ -0,0 +1,54 @@
+// Package chi provides tracing functions for tracing the go-chi/chi package (https://github.com/go-chi/chi).
+package chi // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/go-chi/chi"
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// Middleware returns middleware that will trace incoming requests.
+func Middleware(service string) func(next http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		fn := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
We don't need this variable, we can just return here directly, casting to `http.HandlerFunc`

Reply:
This is the way it's been written in most of the chi middleware. Kept it this way to maintain consistency!

Reply:
As you wish, I don't have a strong feeling about this.